thriftbp/server_test.go

```diff
@@ -0,0 +1,62 @@
+package thriftbp_test
+
+import (
+	"context"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/reddit/baseplate.go"
+	"github.com/reddit/baseplate.go/thriftbp"
+)
+
+func ExampleNewServerFromOptions() {
```

Comment:
please add a doc comment to the example function and some inline comments to explain what you are doing in this example. see https://github.com/reddit/baseplate.go/blob/v0.9.1/thriftbp/example_client_test.go for an example (https://pkg.go.dev/github.com/reddit/baseplate.go@v0.9.1/thriftbp#example-MonitorClient is the result rendered on doc site)